profiles/models.py

```diff
@@ -179,6 +181,11 @@ def import_customer_data(cls, data):
                     profile.phones.create(
                         phone=phone, phone_type=PhoneType.MOBILE, primary=index == 0
                     )
+                ServiceConnection.objects.create(
+                    profile=profile,
+                    service=Service.objects.get(service_type=ServiceType.BERTH),
```

Comment:
Is it a big task to make the `Service` as dropdown in the import form?

Reply:
not sure how necessary that is, but I think the import could be used by any service, at least in theory

Reply:
Not really a huge task, but I would leave it as part of the story where we generalize the whole import. Now this was strictly scoped as `Timmi => Venepaikka` 🤓

Reply:
ok, fair enough 👍